jenkins/bootstrap.py

```diff
@@ -385,10 +385,17 @@ def metadata(repo, artifacts):
     if not meta or not isinstance(meta, dict):
         meta = {}
     meta['repo'] = repo
+
+    try:
+        commit = call(['git', 'rev-parse', '--short=14', 'HEAD^{commit}'], output=True)
```

Comment:
hummm... seems give the same value and Google does not wanna tell me the difference... @ixdy